src/AnalyticsEngine/App.ControlPanel.Engine/SqlExtentions/Profiling-03-CreateSchema.sql

```diff
@@ -1071,7 +1176,14 @@ BEGIN
       [Teams Audio Duration Seconds], [Teams Video Duration Seconds], [Teams Screenshare Duration Seconds],
       [Teams Post Messages], [Teams Reply Messages], [Teams Urgent Messages],
 
-      [Yammer Posted], [Yammer Read], [Yammer Liked]
+      [Yammer Posted], [Yammer Read], [Yammer Liked],
+
+      -- Add here Copilot fields
+
+      -- TODO
+      [Copilot ChatsCount], [Copilot MeetingsCount], [Copilot FilesCount]
```

Comment:
Please use separate words for the metric names. Makes them friendlier and they don't need to be changed later in the report.